tools/lsp/language.rs

```diff
@@ -830,6 +842,39 @@ fn get_code_actions(
             }),
             ..Default::default()
         }));
+    } else if token.kind() == SyntaxKind::Identifier
+        && node.kind() == SyntaxKind::QualifiedName
+        && node.parent().map(|n| n.kind()) == Some(SyntaxKind::Element)
+        && has_experimental_client_capability(client_capabilities, "snippetTextEdit")
+    {
+        let r = map_range(&token.source_file, node.parent().unwrap().text_range());
+        let element = element_at_position(document_cache, &uri, &r.start);
+        let element_indent = element.as_ref().and_then(find_element_indent);
+        let indented_lines = node
+            .parent()
+            .unwrap()
+            .text()
+            .to_string()
+            .lines()
+            .map(|line| format!("    {}", line))
+            .collect::<Vec<String>>();
+        let edits = vec![TextEdit::new(
+            lsp_types::Range::new(r.start, r.end),
+            format!(
+                "${{0:element}} {{\n{}{}\n}}",
+                element_indent.unwrap_or("".into()),
+                indented_lines.join("\n")
+            ),
+        )];
+        result.push(CodeActionOrCommand::CodeAction(lsp_types::CodeAction {
+            title: "Wrap in element...".into(),
```

Comment:
```suggestion
            title: "Wrap in element".into(),
```

Not sure what the `...` was supposed to mean. It looks superflous to me.  It feels like this may ask for more options, but it just apply the edit.